### DISCUSSION TITLE
What is the recommended way of overriding LSP keymaps?

### DISCUSSION THREAD

**Discussion by knpwrs** [LazyVim/LazyVim#1264] [Q&A] [answered]
Given that these aren't tied to a particular plugin (I'm new to lazyvim so I may be reading this incorrectly): https://github.com/LazyVim/LazyVim/blob/566049aa4a26a86219dd1ad1624f9a1bf18831b6/lua/lazyvim/plugins/lsp/keymaps.lua#L14-L49

What is the recommended way to override these keymaps? Say if I wanted `<Leader>le` to be my code action map, for instance.

**Comment by dpetka2001** [comment, marked as answer]
[docs](https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps)?

**Reply by knpwrs** [reply]
I completely missed that part of the docs! Thank you!
